midealocal/devices/ac/__init__.py

```diff
@@ -62,15 +62,17 @@ class DeviceAttributes(StrEnum):
 
 
 class MideaACDevice(MideaDevice):
-    _fresh_air_fan_speeds = {
+    _fresh_air_fan_speeds: ClassVar[dict[int, str]] = {
         0: "Off",
         20: "Silent",
         40: "Low",
         60: "Medium",
         80: "High",
         100: "Full",
     }
-    _fresh_air_fan_speeds_rev = dict(reversed(_fresh_air_fan_speeds.items()))
+    _fresh_air_fan_speeds_rev: ClassVar[dict[str, int]] = {
+        k: v for v, k in _fresh_air_fan_speeds.items()
+    }

```

Comment:
This doesn't seem right. _fresh_air_fan_speeds_rev should be:
{
100: "Full",
80: "High",
...
0: "Off",
}

Reply:
Is now removed